docs/instructions.md

```diff
@@ -75,10 +74,44 @@ After following the steps in [Quick Start](#quick-start-app) above:
 
 ### App Settings ###
 
+#### iOS
+
+To change your general settings for the Whisper app, click the Settings button on the start screen. You will be automatically redirected to the Whisper section of your device's Settings app, which contains two lists of options. The first, "Allow Whisper to Access," lets you control how the app interacts with other features on your device. The second, "Whisper Settings," lets you control how the app itself behaves. 
+
+**Allow Whisper to Access** has four sections:
+
+* **Bluetooth** — allowing Whisper access to Bluetooth is what lets you start and join in-person conversations automatically when you tap Whisper or Listen. We highly recommend allowing Bluetooth access!
+* **Siri & Search** — tapping this option brings you to a menu where you can refine how much access Apple has to how you use Whisper, as well as whether you can use the built-in voice command functions to find and open Whisper
+* **Notifications** — tapping this option brings you to a menu where you can change what notifications (if any) the Whisper app will give you, the style of those notifications, and whether you'll see whispers on your lock screen
```

Comment:
We recommend that Listeners allow notifications because otherwise, when the app is in the background, Listeners will miss that the Whisperer is trying to get their attention (by playing a sound).

Reply:
Will just add slight rewording of this to the existing line